ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -36,13 +37,21 @@ ScrollView {
             //% "Notification preferences"
             text: qsTrId("notifications-preferences")
             anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.leftMargin: Style.current.padding
+            anchors.right: parent.right
+            anchors.rightMargin: Style.current.padding
```

Comment:
Yeap, it's just that in this case, the parent already has the correct padding, but all sections, except for one, need to have some more padding (that's how you get the effect that the selected message preview box goes wider than the rest of the content outline.

I could wrap all sections in another layer of parents though to avoid repetition..

Reply:
Tried it using `Item` around sections, but then, it seems that `Column`s don't populate their height. Leaving as is for now

Reply:
> all sections, except for one, need to have some more padding

You can always do the trick where they all have 0 margin, but the one element you need to be bigger you put negative margins. As long as the parent doesn't have `clip: true`, the Item will grow out of the parent's padding